src/SFML/Graphics/RenderStates.cpp

```diff
@@ -35,13 +35,18 @@ namespace sf
 ////////////////////////////////////////////////////////////
 // We cannot use the default constructor here, because it accesses BlendAlpha, which is also global (and dynamically
 // initialized). Initialization order of global objects in different translation units is not defined.
-const RenderStates RenderStates::Default(BlendMode(
-    BlendMode::SrcAlpha,
-    BlendMode::OneMinusSrcAlpha,
-    BlendMode::Add,
-    BlendMode::One,
-    BlendMode::OneMinusSrcAlpha,
-    BlendMode::Add));
+// For similar reasons we need to provide the default stencil mode, and thus all other settings along with it
```

Comment:
Is that true? I don't get it.

Reply:
No it's just a leftover from the initial implementation, will be removed.